### ISSUE TITLE
Choosing `e1` and `e2`

### ISSUE THREAD

**Issue by luc4leone** [Pomax/BezierInfo-2#326]
[Section 31](https://pomax.github.io/bezierinfo/#pointcurves)

>There are again all kinds of approaches we can take, and a simple-but-effective one is to set the length of that segment to "one third the length of the baseline"

1. What's the **baseline**?
2. Why is it a good approach? 

>This forces e1 and e2 to always be the "linear curve" distance apart, which means if we place our three points on a line, it will actually look like a line.

Totally lost here :) @Pomax can you please elaborate more or in a different way?

**Comment by Pomax** [comment]
The base line is the line between start and end.

As for (2) and the followup question: draw it out to understand it. Take a cubic curve, place all its control points on a line, equally spaced, so for instance start=(0,0), c1=(1,0), c2=(2,0), end=(3,0). This is still a Bezier curve, but it _draws_ a line. Now, compute and draw points for t=0, t=1/3, t=2/3, and t=1 on that curve: where are they?

**Comment by luc4leone** [comment]
>what's special about their where they ended up?

I am not sure I get it. What I see is that **they stay on the line** and that **they are always 4 grid space apart**. Is that what I should have come to? Still don't understand why setting `e2-e1` == "one third the length of the baseline" is a good approach. Still need your help 😉 

## t = 0
![image](https://user-images.githubusercontent.com/2797254/131705464-9ea8070e-b155-44fb-9db7-2776aaade570.png)
## t = 1/3
![image](https://user-images.githubusercontent.com/2797254/131704757-8c6a7448-be2f-452e-93e1-45f5dc5b61c7.png)
## t == 2/3
![image](https://user-images.githubusercontent.com/2797254/131705201-9d918f46-1bda-42c0-9f34-f8d6d4820e73.png)
## t = 1
![image](https://user-images.githubusercontent.com/2797254/131705379-b1c385f9-0cc2-4eb6-905b-bf26dd3398ba.png)

**Comment by Pomax** [comment]
Good start, now, do the same thing but move the control points up one grid spot, so you can see the construction skeleton. Now what do you notice about where e1 and e2 are?

**Comment by luc4leone** [comment]
![image](https://user-images.githubusercontent.com/2797254/131801728-3a966a17-ef65-43b7-8006-3dc2d766bc2e.png)

>Now what do you notice about where e1 and e2 are?

That they are on the same x of the control points? And that they are still 1/3 of the baseline away, comparing to when all the control points were in the same line.

**Comment by Pomax** [comment]
right, so now we know why "one third of the baseline" is at least a good approach: it matches how cubic curves already work.

**Comment by luc4leone** [comment]
more clear now @Pomax thanks, but not totally. Maybe by compare & contrast I'll get it. Can you give an example of another reasonable approach to choose the length of the segment `e2-e1`; instead of "1/3 of the baseline" what would be another good one?

**Comment by Pomax** [comment]
I can't because it depends entirely on your own aesthetic choices and the needs of your users, and what you consider "a good looking curve". You work backwards: you have three points, what curve would you draw through that? Cool, now what cubic curve parameters go with that?

**Comment by luc4leone** [comment]
ok, got it. I'll so some experiments, thanks!
